src/main/kotlin/no/nav/syfo/identhendelse/database/IdenthendelseQuery.kt

```diff
@@ -0,0 +1,132 @@
+package no.nav.syfo.identhendelse.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.dialogmotekandidat.database.PDialogmotekandidatEndring
+import no.nav.syfo.dialogmotekandidat.database.PDialogmotekandidatStoppunkt
+import no.nav.syfo.dialogmotestatusendring.database.PDialogmoteStatus
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.unntak.database.domain.PUnntak
+import java.sql.ResultSet
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryUpdateKandidatStoppunkt =
+    """
+        UPDATE DIALOGMOTEKANDIDAT_STOPPUNKT
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun DatabaseInterface.updateKandidatStoppunkt(nyPersonident: PersonIdentNumber, stoppunktWithOldIdent: List<PDialogmotekandidatStoppunkt>): Int {
+    var updatedRows = 0
+    this.connection.use { connection ->
+        connection.prepareStatement(queryUpdateKandidatStoppunkt).use {
+            stoppunktWithOldIdent.forEach { stoppunkt ->
+                it.setString(1, nyPersonident.value)
+                it.setString(2, stoppunkt.personIdent.value)
+                it.executeUpdate()
```

Comment:
Bedre å gjøre 
`updatedRows += it.executeUpdate()`

Reply:
Husk at det kan være mange rader for hver personIdent her, så må kanskje tenke på om det er antall rader oppdatert som er interessant, eller antall identer.

Tror det gjelder alle de 4 tabellene som skal oppdateres i denne app'en.

Reply:
Hmm ja jeg har vel så langt tenkt at det er antall rader, men det kunne vi jo diskutert på nyåret. Det er bare et tall vi logger per nå. Det er bare `isdialogmote` som har dette "på" i prod nå, så ikke for sent å endre.